source/base/function_lib_cutoff.cc

```diff
@@ -72,7 +229,7 @@ namespace Functions
   {
     if (this->selected == CutOffFunctionBase<dim>::no_component ||
         component == this->selected)
-      return ((this->center.distance(p) < this->radius) ? 1. : 0.);
+      return ((this->center.distance(p) < this->radius) ? this->rescaling : 0.);
```

Comment:
Does this change the value returned? In other words, is this backward compatible?

Reply:
No. The default constructor takes `false` as default value for `integrate_to_one`. That flag is checked in order to set `rescaling`, and if it is false, `rescaling` is always set to 1.